resources/shaders/rt_hybrid.hlsl

```diff
@@ -274,17 +307,15 @@ void ReflectionHit(inout ReflectionHitInfo payload, in MyAttributes attr)
     float3 kD = 1.0 - kS;
     kD *= 1.0 - metal;
 
-	float3 specular = (float3(0, 0, 0)) * F;
-	float3 diffuse = albedo * sampled_irradiance;
+	float3 specular = reflection * F;
+	float3 diffuse = lighting * sampled_irradiance;
 	float3 ambient = (kD * diffuse + specular);
 
-	float3 lighting = shade_pixel(hit_pos, V, albedo, metal, roughness, fN, payload.seed, 1);
-
-	payload.color = ambient + lighting;
+	// Output the final reflections here
+	payload.color = ambient;
```

Comment:
Return reflections. Don't all the lighting information.

Reply:
See previous comment.